### ISSUE TITLE
BLE Library Compilation Error with Arduino as ESP-IDF component

### ISSUE THREAD

**Issue by coolbreeze413** [espressif/arduino-esp32#2866]
### Hardware:
Board:							ANY

### Description:
Compilation Error in BLE_Advertising.cpp at lines 265 and 274:

```
log_v(">> setPrivateAddress")
log_v("<< setPrivateAddress")
```

https://github.com/espressif/arduino-esp32/blob/master/libraries/BLE/src/BLEAdvertising.cpp#L265

https://github.com/espressif/arduino-esp32/blob/master/libraries/BLE/src/BLEAdvertising.cpp#L274

Both are missing semicolons!

Introduced as part of commit:
https://github.com/espressif/arduino-esp32/commit/bea7bd1852bae767864dbb582ea4d4123bc968c1

**Comment by atanisoft** [comment]
@coolbreeze413 please consider closing this issue as this has been fixed via PR #2876

**Comment by coolbreeze413** [comment]
closing the issue, as fix merged.
apologize for delay, missed the merge notification.
